target/scripts/startup/setup.d/security/rspamd.sh

```diff
@@ -294,3 +303,33 @@ function __rspamd__setup_check_authenticated() {
       "${MODULE_FILE}"
   fi
 }
+
+# This function performs a simple check: go through DKIM configuration files, acquire all
+# private key file locations and check whether they exist and whether they can be
+# accesses by Rspamd.
+function __rspamd__check_dkim_permissions() {
+  local DKIM_CONF_FILES DKIM_KEY_FILES
+  [[ -f ${RSPAMD_LOCAL_D}/dkim_signing.conf ]] && DKIM_CONF_FILES+=("${RSPAMD_LOCAL_D}/dkim_signing.conf")
+  [[ -f ${RSPAMD_OVERRIDE_D}/dkim_signing.conf ]] && DKIM_CONF_FILES+=("${RSPAMD_OVERRIDE_D}/dkim_signing.conf")
+
+  # Here, we populate DKIM_KEY_FILES which we later iterate over. DKIM_KEY_FILES
+  # contains all keys files configured by the user.
+  local FILE
+  for FILE in "${DKIM_CONF_FILES[@]}"; do
+    readarray -t DKIM_KEY_FILES_TMP < <(grep -o -E 'path = .*' "${FILE}" | cut -d '=' -f 2 | tr -d ' ";')
+    DKIM_KEY_FILES+=("${DKIM_KEY_FILES_TMP[@]}")
+  done
+
+  for FILE in "${DKIM_KEY_FILES[@]}"; do
+    if [[ -f ${FILE} ]]; then
+      __rspamd__log 'trace' "Checking DKIM file '${FILE}' now"
+      if __do_as_rspamd_user cat "${FILE}" &>/dev/null; then
+        __rspamd__log 'trace' "DKIM file '${FILE}' permissions and ownership appear correct"
+      else
+        __rspamd__log 'warn' "Rspamd DKIM private key file '${FILE}' does not appear to have correct permissions/ownership for Rspamd to use it - please correct permissions/ownership"
```

Comment:
```suggestion
      else
        __rspamd__log 'warn' "Rspamd DKIM private key file '${FILE}' does not appear to have correct permissions/ownership for Rspamd to use it"
```

I think the last portion is redundant. This message also highlights the prior review comment concern, the correct permissions / ownership is not communicated to the user either. Ideally they don't have to think about that with the "make an internal copy" approach (_while we as maintainers also benefit with a predictable DKIM keypair location for both opendkim and rspamd to use?_)